components/_patterns/02-molecules/address-bar/address-bar~04-full.yml

```diff
@@ -0,0 +1,24 @@
+address_bar__address: '515 Malcolm X Blvd, New York, NY 10037'
+address_bar__phone_number: '555-123-4567'
+days:
+  - day: 'Monday'
+    hours: '7:00AM - 7:00PM'
```

Comment:
Can we add a space between `7:00` and `AM`, `7:00` and `PM` for these items? According to last I heard from @jackkim9 this is how our Communications department prefers that we display hourly information.

Reply:
Yes. Should there also be a space there for the date bar? Like this:
<img width="229" alt="Screen Shot 2019-09-10 at 6 26 06 AM" src="https://user-images.githubusercontent.com/1663810/64610018-e52b8a80-d393-11e9-8d5f-a83cbf8be6bf.png">

Reply:
Yep! (As far as I understand it.)